{{cookiecutter.project_slug}}/Makefile

```diff
@@ -49,6 +49,7 @@ clean-test: ## remove test and coverage artifacts
 
 lint: ## check style with flake8
 	flake8 {{ cookiecutter.project_slug }} tests
+	black --check --target-version py36 {{ cookiecutter.project_slug }} tests
```

Comment:
this black line simply warn or will re-format the code?

Reply:
This is simply a warning. Linting code shouldn't reformat it.